lib/mvi/service.rb

```diff
@@ -110,7 +110,7 @@ def measure_info(user_identity)
     end
 
     def get_mvi_error_key(e)
-      error_name = e.body&.first&.[](:displayName)
+      error_name = e.body[:other]&.first&.[](:displayName)
```

Comment:
Just looking at the diff....it looks like before `e.body` was not guaranteed to exist.  Is it now?
Should this be?  
```suggestion
      error_name = e.body&.[](:other)&.first&.[](:displayName)
```

Reply:
That is correct!